exercises/swift-scheduling/canonical-data.json

```diff
@@ -0,0 +1,153 @@
+        "meetingStart": "1997-04-29T10:50:00",
+        "description": "EOW"
+      },
+      "expected": "1997-05-02T17:00:00"
+    },
+    {
+      "uuid": "4afbb881-0b5c-46be-94e1-992cdc2a8ca4",
+      "description": "EOW on Wednesday translates to Friday at five in the afternoon",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2005-09-14T11:00:00",
+        "description": "EOW"
+      },
+      "expected": "2005-09-16T17:00:00"
+    },
+    {
+      "uuid": "e1341c2b-5e1b-4702-a95c-a01e8e96e510",
+      "description": "EOW on Thursday translates to Sunday at eight in the evening",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2011-05-19T8:30:00",
+        "description": "EOW"
+      },
+      "expected": "2011-05-22T20:00:00"
+    },
+    {
+      "uuid": "bbffccf7-97f7-4244-888d-bdd64348fa2e",
+      "description": "EOW on Friday translates to Sunday at eight in the evening",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2022-08-05T12:00:00",
+        "description": "EOW"
+      },
+      "expected": "2022-08-07T20:00:00"
+    },
+    {
+      "uuid": "439bf09f-3a0e-44e7-bad5-b7b6d0c4505a",
+      "description": "2M before the second month of this year translates to the first workday of the second month of this year",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2007-01-02T14:15:00",
+        "description": "2M"
+      },
+      "expected": "2007-02-01T08:00:00"
+    },
+    {
+      "uuid": "86d82e83-c481-4fb4-9264-625de7521340",
+      "description": "11M in the eleventh month translates to the first workday of the eleventh month of next year",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2013-11-21T15:30:00",
+        "description": "11M"
+      },
+      "expected": "2014-11-03T08:00:00"
+    },
+    {
+      "uuid": "0d0b8f6a-1915-46f5-a630-1ff06af9da08",
+      "description": "4M in the ninth month translates to the first workday of the fourth month of next year",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2019-11-18T15:15:00",
+        "description": "4M"
+      },
+      "expected": "2020-04-01T08:00:00"
+    },
+    {
+      "uuid": "06d401e3-8461-438f-afae-8d26aa0289e0",
+      "description": "Q1 in the first quarter translates to the last workday of the first quarter of this year",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2003-01-01T10:45:00",
+        "description": "Q1"
+      },
+      "expected": "2003-03-31T08:00:00"
+    },
+    {
+      "uuid": "eebd5f32-b16d-4ecd-91a0-584b0364b7ed",
+      "description": "Q4 in the second quarter translates to the last workday of the fourth quarter of this year",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2001-04-09T09:00:00",
+        "description": "Q4"
+      },
+      "expected": "2001-12-28T08:00:00"
```

Comment:
My implementation says 2001-12-31 is the last workday of this quarter.

Reply:
Correct, fixed